tools/accuracy_checker/openvino/tools/accuracy_checker/annotation_converters/cvat_hands_and_palm.py

```diff
@@ -0,0 +1,193 @@
+        for point in image_annotation:
+            idx = int(point.get('category_id')) - 1
+            keypoints = point.get('keypoints')
+            assert len(keypoints) >= 2
+            landmarks_x[idx] = float(keypoints[0])
+            landmarks_y[idx] = float(keypoints[1])
+
+        return np.array(landmarks_x), np.array(landmarks_y)
+
+    @staticmethod
+    def generate_labels_mapping(categories):
+        label_to_id = {t['name']: t['id'] for t in categories}
+        meta = {'label_map': {value: key for key, value in label_to_id.items()},
+                'label': list(label_to_id.keys()),
+                'wrist_id': [t['id'] for t in categories if t['name'] == 'WRIST'][0] - 1,
+                'mf_mcp_id': [t['id'] for t in categories if t['name'] == 'MIDDLE_FINGER_MCP'][0] - 1
+                }
+        return label_to_id, meta
+
+
+class CVATHandLandmarkConverter(FileBasedAnnotationConverter, CVATHandPalmConverterBase):
+    __provider__ = 'cvat_hand_landmark'
+
+    annotation_types = (HandLandmarksAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update({
+            'images_dir': PathField(is_directory=True, optional=True,
+                                    description='path to dataset images, used only for content existence check'),
+            'bbox_file': PathField(is_directory=False, optional=False,
+                                   description='path to file with palm bounding box data'),
+            'from_landmarks': BoolField(optional=True, default=False,
+                                        description='acquire bounding box data from landmarks'),
+            'padding': NumberField(optional=True, default=10, value_type=int,
+                                   description='additional padding while acquiring bounding box data from landmarks'),
+        })
+        return configuration_parameters
+
+    def configure(self):
+        super().configure()
+        self.images_dir = self.get_value_from_config('images_dir') or self.annotation_file.parent
+        self.bbox_file = self.get_value_from_config('bbox_file')
+        self.from_landmarks = self.get_value_from_config('from_landmarks')
+        self.padding = self.get_value_from_config('padding')
+
+    def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
+        annotation = read_json(self.annotation_file)
+        bboxes = read_json(self.bbox_file)
+        _, meta = self.generate_labels_mapping(annotation['categories'])
+        num_landmarks = len(meta['label']) - 1
+        annotations = []
+        content_errors = None if not check_content else []
+        for image_id, image in enumerate(annotation['images']):
+            keypoints = [t for t in annotation['annotations'] if t['image_id'] == image['id']]
+            assert len(keypoints) == 21
```

Comment:
do we need such assert?
if converter can be generally reused, i think we should not have restriction on number of keypoints

Reply:
For hand recognition model number of keypoints in annotation is equal to 21. For adapter compatibility should we have number of keypoints defined as optional configuration parameter with default value set to 21 ?

Reply:
I think it can be useful, btw annotation itself does not contains info about number of keypoints?

Reply:
Fixed